src/Symfony/Bridge/Monolog/Handler/ChromePhpHandler.php

```diff
@@ -41,7 +41,7 @@ public function onKernelResponse(FilterResponseEvent $event)
             return;
         }
 
-        if (!preg_match('{\b(?:Chrome/\d+(?:\.\d+)*|Firefox/(?:4[3-9]|[5-9]\d|\d{3,})(?:\.\d)*)\b}', $event->getRequest()->headers->get('User-Agent'))) {
+        if (!preg_match('{\b(?:Chrome/\d+(?:\.\d+)*|HeadlessChrome|Firefox/(?:4[3-9]|[5-9]\d|\d{3,})(?:\.\d)*)\b}', $event->getRequest()->headers->get('User-Agent'))) {
```

Comment:
should this be more specific? ie `preg_match('{\b(?:(?:Headless)?Chrome/\d+(?:\.\d+)*|Firefox...`